inbm/dispatcher-agent/dispatcher/dispatcher_class.py

```diff
@@ -782,12 +782,13 @@ def process_scheduled_requests(scheduled_requests: Sequence[Schedule]):
             logger.debug(f"Scheduled repeated job: {repeated_schedule}")
 
         for imm in schedule.immedate_requests:
+            dispatcher._send_result("", request_id)
             for manifest in imm.manifests:
                 try:
                     dispatcher.do_install(xml=manifest)
                 except (NotImplementedError, DispatcherException) as e:
-                    dispatcher._send_result(str(e), request_id)
-        dispatcher._send_result("", request_id)
+                    # TODO: Save the error for query request
+                    dispatcher._send_result(str(Result(CODE_BAD_REQUEST, str(e))))
```

Comment:
_send_result needs request_id as second parameter

Reply:
(but probably we won't have this line as we are only sending an acknowledgement; error per operation can be queried later)

Reply:
I was thinking the error would be stored in the database. The user can use the query command to check the status. 

Reply:
> I was thinking the error would be stored in the database. The user can use the query command to check the status.

Makes sense, the error should go in the Dispatcher database linked to the job.

Reply:
Reviewing this again this morning, I believe this line `dispatcher._send_result(str(Result(CODE_BAD_REQUEST, str(e))))` should be removed as at this point we have already sent our result.